server/lib/src/constants/acp.rs

```diff
@@ -2064,3 +2067,91 @@ lazy_static! {
         ..Default::default()
     };
 }
+
+// Application Create/Manage
+//   needs to be able to assign to entry managed by
+lazy_static! {
+    pub static ref IDM_ACP_APPLICATION_CREATE_DL6: BuiltinAcp = BuiltinAcp {
+        classes: vec![
+            EntryClass::Object,
+            EntryClass::AccessControlProfile,
+            EntryClass::AccessControlCreate,
+        ],
+        name: "idm_acp_application_create",
+        uuid: UUID_IDM_ACP_APPLICATION_CREATE_V1,
+        description: "Builtin IDM Control for creating new applications.",
+        receiver: BuiltinAcpReceiver::Group(vec![UUID_IDM_APPLICATION_ADMINS]),
+        target: BuiltinAcpTarget::Filter(ProtoFilter::And(vec![
+            match_class_filter!(EntryClass::Application).clone(),
+            FILTER_ANDNOT_TOMBSTONE_OR_RECYCLED.clone(),
+        ])),
+        create_attrs: vec![
+            Attribute::Class,
+            Attribute::Name,
+            Attribute::EntryManagedBy,
+            Attribute::Description,
+        ],
+        create_classes: vec![EntryClass::Object, EntryClass::Application],
+        ..Default::default()
+    };
```

Comment:
minor, but try to space things out. When you have structs stacked without spaces it's hard to follow and separate them.

Reply:
Fixed.